docs/admin/prerequisites/gitlabrunner.rst

```diff
@@ -0,0 +1,65 @@
+.. _gitlabrunner:
+
+Gitlab runner configuration
+===========================
+
+In order for Gitlab to build images that later will be used for launching the Jupyter Notebooks, you need to configure a node that runs a Gitlab runner.
+These are the steps needed for setting up a Gitlab Runner. You can make your own variations of the configuration as you see best fit.
+
+1. Install Docker and Gitlab runner in the VM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+   $ sudo yum update docker-ce --disableexcludes=main # In case you use a centos image with older version of docker
+   $ curl -L https://packages.gitlab.com/install/repositories/runner/gitlab-runner/script.rpm.sh | sudo bash
+   $ sudo yum install gitlab-runner
```

Comment:
why do you need to do this if you are running it with docker?

Reply:
don't need to, leftovers from an old doc